Packages/vcs/Lib/Canvas.py

```diff
@@ -2679,11 +2679,25 @@ def plot_filledcontinents(self,slab,template_name,g_type,g_name,bg,ratio):
         except Exception,err:
             print err
 
+    def __new_elts(self,original,new):
```

Comment:
please approve, I'll do the pep8 anyway and autopep8 does this... No time for this right now. Thanks.